src/config-hardware/webcams.md

```diff
@@ -0,0 +1,23 @@
+# Configuring Webcams
+
+Pop!\_OS will automatically detect most USB webcam models upon connection.
+
+---
+
+## Disabling and Enabling Webcams
+
+You can disable your webcam by pressing `Fn` + `F10`, which disables the webcam USB bus at the device level. Press `Fn` + `F10` to re-enable the webcam, and then restart any application(s) that were using the webcam at the time it was disabled.
+
+## Webcam Compatibility
+
+Pop!\_OS uses the USB Video Class (UVC) device driver for Linux. Devices supported by this driver are [listed on the UVC driver's site](http://www.ideasonboard.org/uvc/).
+
+## Installing Cheese
+
+Test your webcam functionality using [Cheese](https://github.com/GNOME/cheese). Cheese is an open-source webcam application for Linux available for installation in the [Pop!\_Shop](manage-apps/using-pop-shop.ingmd).
```

Comment:
> Test your webcam functionality using [Cheese](https://github.com/GNOME/cheese).

It would be more polite to say:

> You can test your webcam functionality using [Cheese](https://github.com/GNOME/cheese).

Also:

> Cheese is an open-source webcam application

I would call it a `webcam booth application` to better compare it to e.g. Photo Booth on macOS.